src/LeanCode.ContractsGenerator.Tests/ExampleBasedHelpers.cs

```diff
@@ -29,7 +29,7 @@ public static AssertedExport ProjectsCompile(params string[] paths)
     {
         var projectPaths = paths.Select(p => Path.Join("examples", p));
         // HACK: The sync execution results in much cleaner tests
-        var compiled = ContractsCompiler.CompileProjectsAsync(projectPaths).Result;
```

Comment:
TBH that's a totally unnecessary change :P

Reply:
Right, I guess I'll revert it. It's just because `Task.GetAwaiter().GetResult()` doesn't wrap exceptions like `Task.Result` does so I thought it would make sense to catch the actual error cause in tests instead of always getting `AggregateException`.

Reply:
Let's leave the change you made, although it's not necessary, it's a nice addition but I couldn't hold myself back :P